tests/BranchProtection.py

```diff
@@ -27,22 +27,19 @@ class BranchProtection(Framework.TestCase):
     def setUp(self):
         super().setUp()
         self.branch_protection = (
-            self.g.get_user()
-            .get_repo("PyGithub")
-            .get_branch("integrations")
+            self.g
+            .get_repo("curvewise-forks/PyGithub")
+            .get_branch("master")
             .get_protection()
         )
 
     def testAttributes(self):
-        self.assertTrue(self.branch_protection.required_status_checks.strict)
-        self.assertEqual(
-            self.branch_protection.required_status_checks.contexts, ["foo/bar"]
-        )
```

Comment:
Yes, it would be great if you could add a branch protection rule like this to keep these assertions.

Reply:
Completed in c9a9fec8c54ff2512d686b46b25c640b54506d14